polars/polars-io/src/csv/buffer.rs

```diff
@@ -424,11 +424,26 @@ where
     };
 
     match &buf.compiled {
-        Some(compiled) => {
-            match DatetimeInfer::<T::Native>::try_from(compiled.pattern_with_offset.pattern) {
+        Some(compiled) => match DatetimeInfer::<T::Native>::try_from(compiled.pattern) {
```

Comment:
this file is much easier to review with the "hide whitespace" option, the diff essentially boils down to
```diff
-                    infer.utc = true;
-                    let parsed = infer.parse(val, None);
+                let parsed = infer.parse(val);
```
, the rest is just indentation